core/renderers/zelos/drawer.js

```diff
@@ -143,8 +150,31 @@ Blockly.zelos.Drawer.prototype.drawFlatBottom_ = function() {
  * @override
  */
 Blockly.zelos.Drawer.prototype.drawInlineInput_ = function(input) {
-  // Don't draw an inline input.
   this.positionInlineInputConnection_(input);
+
+  var hasConnection = !!input.connection.targetConnection;
```

Comment:
What's the case when an inline input doesn't have a connection?

Reply:
Unlike geras we don't want to keep around these outlines if we have a connection.

Reply:
Got it.